### DISCUSSION TITLE
Goal 4: Ensure inclusive and equitable quality education and promote lifelong learning opportunities for all

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#26] [General]
Chat about anything related to Goal 4: [Ensure inclusive and equitable quality education and promote lifelong learning opportunities for all](https://github.com/Project-Resilience/platform/blob/main/goals/goal_04.md)

**Comment by Steven-Birkinbine** [comment]
Draft Project Proposal to be expanded upon:

Initial Questions: How does AI impact Human Development and Learning? How can we use AI to bring quality education into the classroom, personalizing the experience for the learner and helping people learn new skills rapidly? How can we increase the participation of the workforce in the trades through education?


Assumptions

- Human instructors are required
- AI will improve instructor effectiveness
- AI can remove administrative burden from classrooms
- AI can assist TAs
- Instructors will want the additional technology assistance
- AIs can be provided to each student in tech-heavy countries due to individual iPads/Tablets being distributed within the class
- Turnaround time is semester by semester for analysis
- The majority of future jobs will require some level of technology integration

Questions

- Who/what are the Personas and what are their requirements? - Instructor, Adult Student (undergrad), Adult Student (graduate)
- How does the AI know when the student is ready to advance?
- How can AI assist instructors but not replace?
- Will TAs and Instructors be able to effectively teach their students how to speak to the AI?
- What else do instructors need / where else do they need assistance? What are their pain points?
- How can tech/AI be brought to countries/communities that have unreliable energy/internet?

Measures

- Cognitive: Use AI to measure the level of understanding of written essays using NLP
- Cognitive: Automatic grading of tests and gamification via instant feedback on results
- Cognitive: Raw grade results compared across multiple schools/states
- Non-Cognitive: Utilization of Social Emotional Learning (SEL) and Situational Judgement Test (SJT) frameworks, measured over time
- Non-Cognitive: Masked results of on-campus psychological evaluations of students by salaried psychologists, over time

Noise

- Students not assigned to an AI classroom may use AI in their personal life and for school related activities
- Instructors may teach classes slightly differently each semester (should apply to both Control and Test)
- TAs may change each semester (should apply to both Control and Test)
- Unknown and uncontrolled student background (should apply to both Control and Test)

Possible points of Comparison

- Effects of AI tailored training vs Control of No tailored training
- AI available in the classroom to answer questions with Instructor/TA vs Instructor/TA only
- Accelerated learning (AP) classes with AI vs no AI
- Effects on non-cognitive skills for students exposed to AI vs no AI
- Effects on non-cognitive skills for students specifically assigned to work on their non-cognitive skills using AI, vs no AI
- Effectiveness of AI in trade schools compared to effectiveness in non-trade schools
- Effectiveness of classrooms with more essay requirements vs classrooms with multiple choice (using NLP to grade essays)
- Effectiveness of students immediate access to AI pre-grading on their learning, vs no pre-grading

Research Methodology

- Mixed Methods: Quantitative of direct data gathered from the usage of AI & results of SEL/SJT. Qualitative through surveys

Existing relevant research

- David Game College in London, UK, opened its first teacherless classroom in September 2024
- Shoes were given away for free in Kenya at schools to incent students to attend school by creating a positive association for students and parents
- Reference NIST AI Risk Management Framework & EU AI Act to ensure the effort complies with local laws and regulations - https://www.nist.gov/itl/ai-risk-management-framework | https://digital-strategy.ec.europa.eu/en/policies/regulatory-framework-ai
- The use of technology in learning is not beneficial for technology-anxious individuals - https://www.mdpi.com/2079-9292/11/22/3662
- Current AIs are not prepared to replace human educators - https://www.mdpi.com/2076-3417/13/11/6716
- Potential AI tasks include: Creating Handouts, Automated Grading, Creating Graphical or Written Materials, Functioning as a TA, Animating Motionless Pictures, Gamification of Materials, etc. - https://jai.in.ua/archive/2023/2023-2-1.pdf
- Most existing research is focused on the Cognitive Application of AI according to Blooms Taxonomy, i.e. the focus on knowledge acquisition, comprehension, analysis, and evaluation (thinking and processing) with little research in emotions, attitudes, and social skills (feelings and behaviors)
- Preliminary research into non-cognitive skills shows decreased student engagement when AI is included in the classroom, though the research was not rigorous and is in preprint - https://www.preprints.org/manuscript/202310.1741/v1

Sponsors/Partners

- Universities who are interested in using bleeding edge technology in their classroom and may be pursuing AI research already (such as David Game College)

Disclosure

- This is part of a DBA research thesis for Steven Birkinbine, contributors are welcome

**Comment by Steven-Birkinbine** [comment]
*Draft Project Proposal*

# The Effects of AI on Human Learning

## Decision Makers
- School Boards
- Parents
- Teacher Unions

## Objectives
- To improve the ability of humans to learn more quickly at an earlier age and advance the baseline level of knowledge of societies

## Deliverables
- User Interface
- A purpose built AI model trained on academic information that understands natural language and can interpret fundamental cognitive skills through speech patterns and written essays
- Publication(s) in 3*+ journals

## Data Attributes

### Context
- Region: North America and Europe
- Current time spent learning
- Current topics spent studying
- Current academic performance

### Actions
- Gather student feedback on effectiveness of learning
- Gather student feedback on emotional state while learning

### Outcomes
- Speed and quality of learning and development
- Time spent learning
- Depth and breadth of topics covered
- Grades
- STEM Skill Performance
- Critical Thinking

## Data
- None

## Code
- None

# Needs
- Purpose built AI

## Project Summary

### Tier 1

| Section                | Content                                                                                  |
|------------------------|------------------------------------------------------------------------------------------|
| **Project Objective**  | Improve human learning and development through the utilization of AI in classrooms                                                             |
| **Current Practice**   | Human instructors, humans teaching humans                   |
| **Innovation**         | Using AI as a supplementary tool to teach humans. AI teaches cognitive functions while humans teach social functions                |
| **Stakeholders**       | Students, Instructors, Parents, Schools                                                                       |
| **Personas**           | Students, Instructors, Parents, Schools |
| **Risks**              | Humans exposed to AI may not learn as effectively as the well tested existing methods                                                                     |
| **Budget**             | Unknown                                                           |
| **Timeline**           | Unknown                                                               |
| **Data Safety**        | All information will be anyonymized. No PII will be collected |

### Tier 2

| Section                     | Content                                                                                  |
|-----------------------------|------------------------------------------------------------------------------------------|
| **Scalable**                | If successful, this can be scaled to any/all schools with access to technology             |
| **Measurable**              | Students will be measured on their ability to perform cognitive tasks and their social skills. Cognitive tasks will weight accuracy above speed. Cognitive tasks will be represented through grades and NLP on essays. Critical thinking ability will be measured through essays. Social skills will be measured using SEL frameworks, self-report assessments, and the SSRS             |
| **Integratable**            | Many schools already provide technology to students. The integration would come in getting school boards on board with using the technology in their classrooms                        |
| **Repeatable**              | Can be repeated as long as school boards are willing to take on the challenge   |
| **Standardized**            | This should be standardized through a similar UI for each student to ensure a similar experience. The AI will judge success of the student through time it takes to answer and correctness of the answer and use that to move the student forward, weighting accuracy above speed.  |
| **Persona Needs and Goals** | It intends to help students learn more efficiently, allows instructors to focus on the human element of teaching, parents to have more successful students, and schools to have higher test scores               |
| **ISO Standard**            | ISO 21001, ISO 29994, and ISO 29991       |
| **Predictive Agent**        | A higher use of AI will result in a higher cognitive test score                |
| **Documentation**           | Pending |
| **Milestones**            | Pending                                                                                |

### Tier 3


| Section                   | Content                                                                                                                                       |
|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
| **Team Members**          | Steven Birkinbine (Project Manager), Pending                                                                                               |
| **Team Structure** | Flat structure with a fluid hierarchy |
| **User Journey Mapping**  | Pending   |
| **Persona Feedback**      | Feedback will be gathered initially daily from students and then reduced to weekly intervals to ensure their learning needs come first                                                      |
| **Volunteer Intake**      | Informal. Contact Steven Birkinbine                                                                                  |


## References

- David Game College in London, UK, opened its first teacherless classroom in September 2024
- Reference NIST AI Risk Management Framework & EU AI Act to ensure the effort complies with local laws and regulations - https://www.nist.gov/itl/ai-risk-management-framework | https://digital-strategy.ec.europa.eu/en/policies/regulatory-framework-ai
- The use of technology in learning is not beneficial for technology-anxious individuals - https://www.mdpi.com/2079-9292/11/22/3662
- Current AIs are not prepared to replace human educators - https://www.mdpi.com/2076-3417/13/11/6716
- Potential AI tasks include: Creating Handouts, Automated Grading, Creating Graphical or Written Materials, Functioning as a TA, Animating Motionless Pictures, Gamification of Materials, etc. - https://jai.in.ua/archive/2023/2023-2-1.pdf
- Most existing research is focused on the Cognitive Application of AI according to Blooms Taxonomy, i.e. the focus on knowledge acquisition, comprehension, analysis, and evaluation (thinking and processing) with little research in emotions, attitudes, and social skills (feelings and behaviors)
- Preliminary research into non-cognitive skills shows decreased student engagement when AI is included in the classroom, though the research was not rigorous and is in preprint - https://www.preprints.org/manuscript/202310.1741/v1
- Arizona Board greenlights AI Classroom - https://www.maginative.com/article/no-teachers-just-ai-arizona-approves-bold-new-charter-school/

**Comment by danyoungday** [comment]
## CausalEdu Dataset

Here is a dataset from Microsoft and Eedi education platform for use in causal inference with regards to learning math.
They record sequences of multiple choice quiz answers as well as which math concepts (constructs) relate to each question.
There are 641490 rows in the main dataset where each row is an answer from a student on a quiz.
They also perform A/B testing alongside using expert feedback to form "ground truth" causal relationships between constructs.
[https://github.com/Eedi/CausalEdu](https://github.com/Eedi/CausalEdu)

#### Context:
- Past quizzes and scores
#### Actions:
- Take lesson 1, 2, 3, 4, etc. or 0: don't do a lesson
#### Outcomes:
- Maximize passing the quiz, minimize number of lessons taken

This could be benchmarked against the ground truth causal relationships, or what the authors + experts deemed were the true best lessons to take.

#### Concerns:
- Potentially not enough background info to form context on student
- This may require a time series predictor
